.gitignore

```diff
@@ -0,0 +1,7 @@
+*.js.map

```

Comment:
If you use the inline sourcemaps option you won't end up with .map files, which imho is nice.